app/ui-react/packages/ui/src/Connection/ConnectionCard.tsx

```diff
@@ -95,46 +108,88 @@ export class ConnectionCard extends React.PureComponent<
           />
         )}
         <Card matchHeight={true}>
-          {this.props.menuProps && (
-            <div className="pull-right">
-              <DropdownKebab
-                id={`connection-${this.props.name}-menu`}
-                pullRight={true}
-                title={this.props.menuProps.i18nMenuTitle}
-              >
-                <li role={'presentation'} key={0}>
-                  <Link to={this.props.href} role={'menuitem'} tabIndex={1}>
-                    {this.props.menuProps.i18nViewLabel}
-                  </Link>
-                </li>
-                <li role={'presentation'} key={1}>
-                  <Link
-                    to={this.props.menuProps.editHref}
-                    role={'menuitem'}
-                    tabIndex={2}
+          <Card.Heading
```

Comment:
Making the card heading visible at all times, since I couldn't find reason for us to hide it -- the padding it has is good and makes it look more like the old app, even if it doesn't have a `Technology Preview` banner on it. If this isn't preferable, please let me know.